src/firefly/js/visualize/ui/ImageCenterDropDown.jsx

```diff
@@ -0,0 +1,166 @@
+    // const createMarkerAndMoveToTarget= (fields) => {
+    //     const wp=  parseWorldPt(get(fields,'UserTargetWorldPt'));
+    //     if (!wp) return;
+    //     moveToTarget(fields);
+    //                              todo: the section below might need to be deferred to the plot has moved to the target first
+    //     const cc = CysConverter.make(plot);
+    //     if (cc.pointInPlot(wp)) {
+    //         const title= prettyTarget(wp);
+    //         const drawLayerId = title;
+    //         dispatchCreateMarkerLayer(drawLayerId, title, plotId, true, wp, true);
+    //     }
+    // };
+    //
+
+    const dropDown= (
+        <SingleColumnMenu>
+
+            <ToolbarButton text='Pan by table row' tip='Center selected images position of the highlighted row'
+                           enabled={coordTables}
+                           horizontal={false} key={'pan-table'}
+                           hasCheckBox={true} checkBoxOn={vr.autoScrollToHighlightedTableRow}
+                           onClick={() => dispatchChangeTableAutoScroll(!vr.autoScrollToHighlightedTableRow)}/>
+
+            <DropDownVerticalSeparator useLine={true}/>
+
+
+            <ToolbarButton text='Center on Target' tip='Center on search target'
+                           enabled={hasSearchTarget}
+                           horizontal={false} key={'center-search-target'}
+                           onClick={() => dispatchRecenter({plotId})} />
+
+            <ToolbarButton text='Center on Image' tip='Center on the image'
+                           enabled={Boolean(plot && isImage(plot))}
+                           horizontal={false} key={'center-image'}
+                           onClick={() => dispatchRecenter({plotId, centerOnImage:true})} />
+
+            <DropDownVerticalSeparator useLine={true}/>
+            <FieldGroup style={{display:'flex', fontSize:'10pt', padding:'5px 0 0 0'}}
+                        groupKey='TARGET_DROPDOWN' validatorFunc={null} keepState={true}>
+                <TargetPanel groupKey={'target-move-group'} labelWidth={80}
+                             label={'Center On:'}
+                             showResolveSourceOp={false} showExample={false}/>
+                <div style={{display:'flex', flexDirection:'column', alignItems: 'flex-end'}}>
+                    <CompleteButton style={{paddingLeft:10}} text= 'Go' onSuccess={moveToTarget} fireOnEnter={true}
+                                    dialogId='ExampleDialog' />
+                    {/*                //todo- button to add marker*/}
+                    {/*<CompleteButton style={{paddingLeft:10, marginTop:4}} text= 'Marker' onSuccess={createMarkerAndMoveToTarget}*/}
+                    {/*                dialogId='ExampleDialog' />*/}
+                </div>
+            </FieldGroup>
+            {recentAry.length>0 && <DropDownVerticalSeparator useLine={true}/>}
+            <DropDownSubMenu text={'Recent Positions'} visible={recentAry.length>0}>
```

Comment:
I would like the tooltips for these to show their resolved coordinates, if what was entered was a name.  I am OK if that is done in a separate ticket at a later date.

Reply:
That makes sense and should not take long. I will add it.